dali/base/dasds.cpp

```diff
@@ -1445,6 +1484,11 @@ class CDeltaWriter : implements IThreaded
                     if (semTimedout)
```

Comment:
I think this can be
if (semTimedout && !needToSendSignalToRequestDataIsWritten)
{
   sem.wait(); // Consume the signal that was sent
   needToSendSignalToRequestDataIsWritten = true;
}

Reply:
I think that's true, I'll modify.